support/cas-server-support-oauth-core-api/src/main/java/org/apereo/cas/support/oauth/web/response/accesstoken/response/OAuth20JwtAccessTokenEncoder.java

```diff
@@ -49,13 +49,26 @@ public class OAuth20JwtAccessTokenEncoder implements CipherExecutor<String, Stri
 
     private final String issuer;
 
+    /**
+     * Doing basic check that {@code JWTParser.parse} does to reduce logging of stack traces.
+     * @param tokenId Possibly encoded token.
+     * @return true
+     */
+    private boolean isPossiblyEncoded(final String tokenId) {
+        return tokenId.contains(".");
+    }
+
     @Override
     public String decode(final String tokenId, final Object[] parameters) {
         return FunctionUtils.doAndHandle(() -> {
             if (StringUtils.isBlank(tokenId)) {
```

Comment:
This is semantically inconsistent; if I pass blank, `""`, I get back null. You should likely return the actual value that was passed, which then helps you merge this with `isPossiblyEncoded`